### ISSUE TITLE
Login user

### ISSUE THREAD

**Issue by cchristelis** [Cadasta/cadasta-qgis-plugin#2]
# Problem

Users need to be logged in to the Cadasta to get access to their data.

# Proposed solution

Build a class, that uses the Network mixin to initiate a session to cadasta.

**Comment by meomancer** [comment]
@cchristelis 
is there any account that i could use?
i tried register,and when activate it, it gaves error
```
Not Found

The requested URL /account/activate/MTQ5/4hl-01e89fa18ebe426a434e/ was not found on this server.
```

**Comment by cchristelis** [comment]
just create a new one?

**Comment by meomancer** [comment]
fixed at #10
